key_layout/key_layout

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright 2016 Patrick Haun
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+setxkbmap -query | awk '/layout/{print $2}' | tr -d '\n'
+variant=$(setxkbmap -query | awk '/variant/{print $2}')
+if [ ! -z $variant ]; then
+  echo -n " "
+  echo $variant
```

Comment:
These two lines could be consolidated into `echo " $variant"`, up to you. if you want to change.